catboost/python-package/catboost/core.py

```diff
@@ -5074,3 +5074,31 @@ def _plot_feature_statistics(statistics, feature, max_cat_features_on_plot):
     else:
         fig = _build_binarized_feature_statistics_fig(statistics, feature)
         return [fig]
+
+
+def to_regressor(model):
+    regressor = CatBoostRegressor()
+
+    params = deepcopy(model._init_params)
+    _process_synonyms(params)
+    if 'loss_function' in params:
+        regressor._check_is_regressor_loss(params['loss_function'])
+
+    for attr in ['_object', '_init_params', '_random_seed', '_learning_rate', '_tree_count']:
```

Comment:
This list of attributes is not complete. For example, we loss feature_importance_, _prediction_values_change and _loss_value_change while converting. Maybe, there are some other lost attributes. Check it carefully, please.